array-filter/main.js

```diff
@@ -0,0 +1,25 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna'
+];
+
+const evenNumbers = numbers.filter(num => num % 2 === 0);
+console.log('evenNumbers:', evenNumbers);
+
+const overFive = numbers.filter(num => num > 5);
+console.log('overFive:', overFive);
+
+const startWithE = names.filter(name => name.startsWith('E'));
+console.log('startWithE:', startWithE);
+
+const haveD = names.filter(name => name.includes('d') || name.startsWith('D'));
```

Comment:
What happens if the name has a capital `'D'` somewhere in the middle (for example, `'Tenacious D'`)?